functions/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
```

Comment:
I'm not sure this file is adding any value. The top-level Makefile is probably sufficient.

Reply:
I think this can provide an additional level of abstract to the functions. Do you prefer to only use top-level Makefile?